install-usercss/external.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20">
+  <path d="M4,4h5v2H6v8h8v-3h2v5H4V4z M11,3h6v6l-2-2l-4,4L9,9l4-4L11,3z"></path>
+</svg>
```

Comment:
Missing EOF.
Also, why an external file? You can create SVG elements in-place using `template` element or `$element()`, see examples in the existing code. The advantage is that the embedded SVG allows restyling via CSS.